keystore/filesystem/translator_keystore.go

```diff
@@ -62,13 +116,3 @@ func (store *TranslatorFileSystemKeyStore) GetPrivateKey(id []byte) (*keys.Priva
 	}
 	return &keys.PrivateKey{Value: privateKey}, nil
 }
-
-// GetPeerPublicKey returns other party transport public key.
-func (store *TranslatorFileSystemKeyStore) GetPeerPublicKey(id []byte) (*keys.PublicKey, error) {
-	filename := getConnectorKeyFilename(id)
-	key, err := ioutil.ReadFile(filepath.Join(store.directory, getPublicKeyFilename([]byte(filename))))
-	if err != nil {
-		return nil, err
-	}
-	return &keys.PublicKey{Value: key}, nil
-}
```

Comment:
Mmm... We can remove this because Acra Translator talks to Acra Server via Acra Connector, and thus uses its peer public key. Acra Server also expects that key for inbound connections. `GetPeerPublicKey()` in the base `KeyStore` is implemented like that. Right?

Reply:
Nice catch. It forced me to try to remember why I removed it and why it works now ))
https://github.com/cossacklabs/acra/blob/master/keystore/filesystem/translator_keystore.go#L28 - we can remove it because keystore for translator inherit this function from base server's keystore and should not be overrided. Because AT and AS expects key from connector like <id>.pub. But we need override `GetPrivateKey` method, because translator has own keys with `_translator` prefix